jwst-masterclass/0.1.0/Dockerfile

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Association of Universities for Research in Astronomy
+# Distributed under the terms of the Modified BSD License.
+
+FROM jupyter/scipy-notebook:a238993ad594
+# See https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html
+# for a list of what comes with the base image
+# python version is: 3.6.8
+
+LABEL maintainer="Science Platforms <cbrasseur@stsci.edu>"
+
+# Environment variables
+ENV MKL_THREADING_LAYER="GNU"
+
+# CRDS environment variables
```

Comment:
It might be useful to put the environment variables at the bottom if they aren't used by the Dockerfile building process. This way, if they ever change, we don't have to rebuild the entire rest of the image losing cache. Might not matter here though.